### PR TITLE
chore: renames from first to numberOfItems in Shelves

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

This PR aims to rename the `first` prop in the `ProductShelf` and `CrossSellingShelf` since the CMS.

## How to test it?

1. in faststore.config.js change the workspace to `formspace`
2. `yarn dev` to run locally.
3. check the Cross Selling Shelves in pdp  or `ProductShelf` in homepage using these links (version from `formspace` workspace):

pdp
http://localhost:3000/api/preview?contentType=pdp&documentId=717bc97f-8f19-4744-a52b-7ccac43cc898&versionId=01b9e4f3-4253-42ee-a8a3-96c6bdea25fa

homepage
http://localhost:3000/api/preview?contentType=home&documentId=ad2fd81d-a53c-4281-8d01-a4fc2f274db3&versionId=97a6e029-47a2-4a6d-8276-e2266690cf04

### Starters Deploy Preview

TBD after this PR got merged (depends on cms sync in master workspace after the merge)

### CODE REVIEW

**Review by icazevedo — APPROVED** [review]

**Review by hellofanny — APPROVED** [review]
LGTM!

It's working as expected. 🥳 
In case the user sets 10 products, but only 6 is available, only 6 will be displayed. I think the use is aware of it, right?

image

**Comment by eduardoformiga** [comment]
> LGTM!
> 
> It's working as expected. 🥳 In case the user sets 10 products, but only 6 is available, only 6 will be displayed. I think the use is aware of it, right?
> 
> image

I could add this explanation in the description field of CMS.

File: packages/core/cms/faststore/sections.json [hunk 1]
```diff
@@ -830,17 +830,17 @@
...
           "title": "Title"
         },
-        "first": {
+        "numberOfItems": {
           "type": "integer",
-          "title": "First",
+          "title": "Total number of items",
           "default": 5,
-          "description": "Number of items to display"
+          "description": "Total number of items. The quantity may be smaller if fewer products are returned in the query."
```
Comment (hellofanny) [inline packages/core/cms/faststore/sections.json:843]:
```suggestion
          "description": "Total number of items. The quantity may be smaller if the query returns fewer products."
```

File: packages/core/cms/faststore/sections.json [hunk 2]
```diff
@@ -925,17 +925,17 @@
...
           "type": "string"
         },
-        "items": {
-          "title": "First",
+        "numberOfItems": {
+          "title": "Total number of items",
           "type": "integer",
           "default": 5,
-          "description": "Number of items to display"
+          "description": "Total number of items. The quantity may be smaller if fewer products are returned in the query."
```
Comment (eduardoformiga) [inline packages/core/cms/faststore/sections.json:938]:
```suggestion
          "description": "Total number of items. The quantity may be smaller if the query returns fewer products."
```

**Pull request merged**